plugins/modules/content_export_version.py

```diff
@@ -0,0 +1,167 @@
+    required: false
+    type: int
+  fail_on_missing_content:
+    description:
+      - Fails if any of the repositories belonging to this version are unexportable.
+    required: false
+    type: bool
+  incremental:
+    description:
+      - Export only the content that has changed since the last export.
+    required: false
+    type: bool
+  from_history_id:
+    description:
+      - Export history identifier used for incremental export. If not provided the most recent export history will be used.
+    required: false
+    type: str
+extends_documentation_fragment:
+  - theforeman.foreman.foreman
+  - theforeman.foreman.foreman.organization
+'''
+
+EXAMPLES = '''
+- name: "Export content view version (full)"
+  content_export_version:
+    content_view: RHEL8
+    content_view_version: '1.0'
+    username: "admin"
+    password: "changeme"
+    server_url: "https://foreman.example.com"
+    organization: "Default Organization"
+    destination_server: "airgapped.example.com"
+
+- name: "Export content view version (full) in chunks of 10 GB"
+  content_export_version:
+    content_view: RHEL8
+    content_view_version: '1.0'
+    username: "admin"
+    password: "changeme"
+    server_url: "https://foreman.example.com"
+    organization: "Default Organization"
+    destination_server: "airgapped.example.com"
+    chunk_size_gb: 10
+
+- name: "Export content view version (full) and fail if any repos are unexportable"
+  content_export_version:
+    content_view: RHEL8
+    content_view_version: '1.0'
+    username: "admin"
+    password: "changeme"
+    server_url: "https://foreman.example.com"
+    organization: "Default Organization"
+    destination_server: "airgapped.example.com"
+    fail_on_missing_content: true
+
+- name: "Export content view version (incremental) since the most recent export"
+  content_export_version:
+      content_view: RHEL8
+      content_view_version: '1.0'
+      username: "admin"
+      password: "changeme"
+      server_url: "https://foreman.example.com"
+      organization: "Default Organization"
+      destination_server: "airgapped.example.com"
+      incremental: true
+
+- name: "Export content view version (incremental) since a specific export"
+  content_export_version:
+      content_view: RHEL8
+      content_view_version: '1.0'
+      username: "admin"
+      password: "changeme"
+      server_url: "https://foreman.example.com"
+      organization: "Default Organization"
+      destination_server: "airgapped.example.com"
+      incremental: true
+      from_history_id: 12345
+'''
+
+from ansible_collections.theforeman.foreman.plugins.module_utils.foreman_helper import KatelloAnsibleModule, _flatten_entity
+
+
+class KatelloContentExportModule(KatelloAnsibleModule):
+    pass
+
+
+def main():
+    module = KatelloContentExportModule(
+        foreman_spec=dict(
+            content_view_version=dict(type='entity', scope=['content_view'], search_by='version', required=False),
```

Comment:
this stores the cv version as `content_view_version`, but the controller expects it as `id`, so let's rename it:

```suggestion
            content_view_version=dict(type='entity', scope=['content_view'], search_by='version', flat_name='id', required=False),
```

Reply:
(and this is the reason you've been seeing "Error while performing version on content_exports: The following required parameters are missing: id" the other day: the apidoc says the controller expects the "id" parameter, but nothing was providing that)